### DISCUSSION TITLE
Свойства пациента необходимые лаб группе

### DISCUSSION THREAD

**Discussion by ai-fhir** [fhir-ru/core#76] [Административная РГ]
Диаграмма описывает поля пациента, которые необходимы лабораторной группе и должны быть учтены в базовом профиле

![lab_patient](https://user-images.githubusercontent.com/93438901/152655211-581bd20c-d1dc-46eb-93b0-54f9a25058ea.jpg)

cc @avzviagin

**Comment by eugenekogan-spb** [comment]
Вопрос:
1) Имеются разные карточки пациента в разных МО, в каждой у этого пациента свои документы, это правильно. Но кто поддерживает единую персону? Существует ли она?
2) Мед организация - возможно, не только Provider, но также и страховая.

**Reply by ai-fhir** [reply]
Можем считать, что нам не нужно поддерживать единую сущность пациента для задачи выгрузки результатов лаб исследований
